schemas/common/threshold.cue

```diff
@@ -21,5 +21,6 @@ package common
 
 #thresholds: {
 	default_color?: string
+	max?:           number
```

Comment:
it would be cool to have a description as comment to explain the purpose of this field.

Reply:
yeah good point, max isn't relevant for thresholds in line charts so I moved it to gauge.cue and added a comment with more details